### PR TITLE
Skrell accents, religions, and citizenship updates.

### PR DESCRIPTION
**Pull request by listerla**
Sprites and accent descriptions by Caelphon, and changes approved by Caelphon.

Adds Homeworld, Qerr'malic, Traverse, Aweiji, and Aliose accents, as well as Suur'ka and Kir'gul religions. Removes some lorebreaking accents, citizenships, and religions.

image
image

### CODE REVIEW

**Comment by listerla** [comment]
!review

File: code/modules/background/accent/skrell.dm [hunk 1]
```diff
@@ -1,5 +1,36 @@
...
+	name = ACCENT_HOMEWORLD
+	description = "The typical accent of the Homeworld (or Qerrbalak), is very much more distinct and  enunciated than other variations of Nral'Malic. It puts specific emphasis on a \
+	melodious manner of speaking, their tones often jumping out of the standard human hearing range."
+	tag_icon = "skrell_homeworld"
+
+/datum/accent/skrell/qerrmalic
+	name = ACCENT_QERRMALIC
+	description = "The Qerr'Malic accent is known for being rather soft, even hushed at some points. Because of the moon's tourist industry, it borrows elements from other accents to \
+	form a rather unique mixture that makes specific reference to keeping low tones accessible to non-Skrell, further cementing the overall 'quietness' to other Skrell as their sentences \
+	sound half-finished. "
```
Comment (Geevies) [inline code/modules/background/accent/skrell.dm:17]:
```suggestion
	sound half-finished."
```

File: code/modules/background/accent/skrell.dm [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+
+/datum/accent/skrell/aliose
+	name = ACCENT_ALIOSE
+	description = "It is often noted that those from Aliose have a distinctive shrill to their voices, emphasizing high notes frequently throughout their speech. The Skrell view this \
+	as guttural, mostly due to the resonations felt deep within their throats when attempting to replicate the accent."
+	tag_icon = "skrell_aliose"
+
+/datum/accent/skrell/aweiji
+	name = ACCENT_AWEIJI
+	description = "This type of accent emphasizes on a balance ofhighs and lows. It uses the nasal canal to further produce a harmonic pattern of speech. Idols are often found replicating \
```
Comment (Geevies) [inline code/modules/background/accent/skrell.dm:28]:
```suggestion
	description = "This type of accent emphasizes on a balance of highs and lows. It uses the nasal canal to further produce a harmonic pattern of speech. Idols are often found replicating \
```

**Review by Geevies — CHANGES_REQUESTED** [review]

**Review by Geevies — APPROVED** [review]

**Review by Casper3667 — CHANGES_REQUESTED** [review]
Almost fine, except you accidentally deleted the taj din'akk accent tag.

**Comment by listerla** [comment]
> 
> 
> Almost fine, except you accidentally deleted the taj din'akk accent tag.

ah fuck

**Review by Casper3667 — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**